### ISSUE TITLE
12-hour statistics format 

### ISSUE THREAD

**Issue by computer777** [AdguardTeam/AdguardForAndroid#4604]
I guess this suggestion would fall under cosmetic.

In the statistics - recent activity page the log is only readable in 24hr format.

Would it be possible to add an option to ‘show in 12hr’ , also?

**Comment by Versty** [comment]
@computer777 Hi! 
Are you talking about minimum statistics period in Adguard For Android?

**Comment by computer777** [comment]
> @computer777 Hi! 
> Are you talking about minimum statistics period in Adguard For Android? 


Yes, I'm sorry. I am talking about the Statistics recent activity page.

It logs inquiries in 24hr format. And, I was curious if a q2hr format could be added. Like, rather than seeing 16:13  it would should 4:13pm.

**Comment by Versty** [comment]
@computer777 Let's see how many votes this feature request will get
